plugins/cost-insights/src/components/CostOverviewCard/CostOverviewChart.tsx

```diff
@@ -135,67 +136,74 @@ export const CostOverviewChart = ({
   };
 
   return (
-    <ResponsiveContainer
-      width={responsive ? '100%' : styles.container.width}
-      height={styles.container.height}
-      className="cost-overview-chart"
-    >
-      <ComposedChart margin={styles.chart.margin} data={chartData}>
-        <CartesianGrid stroke={styles.cartesianGrid.stroke} />
-        <XAxis
-          dataKey="date"
-          domain={['dataMin', 'dataMax']}
-          tickFormatter={overviewGraphTickFormatter}
-          tickCount={6}
-          type="number"
-          stroke={styles.axis.fill}
-        />
-        <YAxis
-          domain={[() => 0, 'dataMax']}
-          tick={{ fill: styles.axis.fill }}
-          tickFormatter={formatGraphValue}
-          width={styles.yAxis.width}
-          yAxisId={data.dailyCost.dataKey}
-        />
-        {metric && (
+    <Box display="flex" flexDirection="column">
+      <CostOverviewLegend
```

Comment:
@timbonicus can't seem to find your previous comment 🤔 I moved `CostOverviewLegend` into its own component since `CostOverviewChart` is getting a bit hairy. I had thought about moving both legends to `CostOverviewCard` but that would also complicate things as there's a bit of calculations and numbers that the cost by product chart + legend both use.